inc/namespace.php

```diff
@@ -902,14 +902,45 @@ function elasticpress_mapping( array $mapping ) : array {
 			$synonyms,
 			$stopwords
 		);
+
 		// Copy default analyzer to default search.
 		$mapping['settings']['analysis']['analyzer']['default_search'] = $mapping['settings']['analysis']['analyzer']['default'];
-		// Add our custom filters.
-		$mapping['settings']['analysis']['analyzer']['default_search']['filter'] = array_merge(
-			array_keys( $synonyms ),
-			array_keys( $stopwords ),
-			$mapping['settings']['analysis']['analyzer']['default_search']['filter']
-		);
+
+		// Add stopwords after `icu_normalizer` if present, otherwise prepend.
+		// This ensures text is lowercased and full-width characters converted to
+		// half-width while still retaining accents. This is should also ensure
+		// stopwords and stemming have not yet been applied.
```

Comment:
Just a nit, but for multiline comments I would use

```
/*
 * Some stuff here
 */
```
...instead of `//` syntax.
🔢